### PR TITLE
lps-83755-with-updated-gradle-plugins

### PR DESCRIPTION
**Pull request by lawrence-lee**
@petershin, here's a pull with the changes you requested from https://github.com/petershin/liferay-portal/pull/176

Please let me know how to proceed with https://github.com/petershin/liferay-portal/pull/176#discussion_r208425864

### CODE REVIEW

**Comment by gamerson** [comment]
ci:test:sf

**Comment by gamerson** [comment]
ci:test:relevant

File: modules/sdk/project-templates/project-templates-theme-contributor/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -78,4 +83,12 @@
 			</plugin>
 		</plugins>
 	</build>
+	<dependencies>
+		<dependency>## ignore dependency comparison
```
Comment (petershin) [inline modules/sdk/project-templates/project-templates-theme-contributor/src/main/resources/archetype-resources/pom.xml]: This is a phrase so it should be capitalized - "## Ignore Dependency Comparison"

For examples see [portal-impl/src/portal.properties](https://github.com/liferay/liferay-portal/blob/master/portal-impl/src/portal.properties)

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:relevant

**Comment by petershin** [comment]
ci:stop

**Comment by petershin** [comment]
Sent to Brian: https://github.com/brianchandotcom/liferay-portal/pull/61999

**Pull request closed without merging**